SuperBuild.cmake

```
@@ -198,6 +198,17 @@ option(USE_SYSTEM_NIFTYREG "Build using an external version of NIFTYREG" OFF)
 option(USE_SYSTEM_GTest "Build using an external version of GTest" OFF)
 option(USE_SYSTEM_ACE "Build using an external version of ACE" ON)
 
+if (USE_SYSTEM_STIR)
+  set(STIR_REQUIRED_VERSION "4.1.0" CACHE STRING ON)
+endif()
```

```suggestion
set(STIR_REQUIRED_VERSION "4.1.0" CACHE STRING ON)
```
nice and simple. However, these lines should go into `version_config.cmake`

---

Of course we can always add the `${proj}_REQUIRED_VERSION` which will be used only in the case we use the system `${proj}`